cwms-data-api/src/main/java/cwms/cda/data/dto/basin/Basin.java

```diff
@@ -0,0 +1,165 @@
+package cwms.cda.data.dto.basin;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.databind.PropertyNamingStrategies;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.annotation.JsonNaming;
+import cwms.cda.api.errors.FieldException;
+import cwms.cda.data.dto.CwmsDTOBase;
+import cwms.cda.data.dto.LocationIdentifier;
+import cwms.cda.formatters.Formats;
+import cwms.cda.formatters.annotations.FormattableWith;
+import cwms.cda.formatters.json.JsonV1;
+
+import java.util.Objects;
+
+@FormattableWith(contentType = Formats.JSONV1, formatter = JsonV1.class, aliases = {Formats.DEFAULT, Formats.JSON})
+@JsonDeserialize(builder = Basin.Builder.class)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonNaming(PropertyNamingStrategies.KebabCaseStrategy.class)
+public final class Basin implements CwmsDTOBase {
+    private final LocationIdentifier basinId;
+    private final Double sortOrder;
+    private final Double basinArea;
+    private final Double contributingArea;
+    private final LocationIdentifier parentBasinId;
+    private final String areaUnit;
+    private final LocationIdentifier primaryStreamId;
+
+    private Basin(Builder builder)
+    {
+        this.basinId = builder.basinId;
+        this.sortOrder = builder.sortOrder;
+        this.basinArea = builder.basinArea;
```

Comment:
Rename to follow DB naming, add "drainage"
TOTAL_DRAINAGE_AREA
CONTRIBUTING_DRAINAGE_AREA

Reply:
Addressed in latest push